middleware/preprocessor.go

```diff
@@ -0,0 +1,26 @@
+package grpc_preprocessor
```

Comment:
I do not really like the package name here. I think `package middleware` will look more consistent here.

Reply:
ok